common/changes/@ducky/plumage/fix-slider-2_2022-08-25-13-06.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@ducky/plumage",
```

Comment:
Please ensure you add a patch changelog for `plumage-react` so that these fixes are released as part of that package.

Reply:
Think I've fixed that now

Reply:
I'm slightly confused by what has been going with your change logs. 
Have you been manually changing the content or using what is generated by `rush change`?  
The final change logs have a type of `none`, is this what you intend? I would not expect so due to the commit message.

Reply:
I manually created the react change log file as it wouldn't generate. Clearly this was a stupid thing to do because I've introduced an error by doing this! I'll ask Tanguy about it and do it the right way.

Reply:
Files are now created automatically. I've set type to patch as I think this is minor update.